PowerShellForGitHub.psd1

```diff
@@ -7,7 +7,7 @@
     CompanyName = 'Microsoft Corporation'
     Copyright = 'Copyright (C) Microsoft Corporation.  All rights reserved.'
 
-    ModuleVersion = '0.16.1'
+    ModuleVersion = '0.17.0'

```

Comment:
Please do not change the module version yourself.  It only gets changed by project administrators at the time that an updated version is released to PowerShellGallery.

Reply:
Reverted